### ISSUE TITLE
Alpha inverted track matte render issue only on iOS.

### ISSUE THREAD

**Issue by ision-industries** [airbnb/lottie-ios#1271]
I’ve been battling this issue now for a few months now and at this point I am stuck. My animation is having an issue rendering on iOS only. Here are some screenshots to give context to my problem. 

This screenshot is from the Android LottieFiles app, as you can see the animation displays correctly. ![image](https://user-images.githubusercontent.com/58711145/95390832-fa8b2500-08c3-11eb-8b66-7f25c0a29e87.jpeg)

This is a screenshot from the LottieFiles app on iOS. As you can see, the alpha inverted matte layer is displaying “blurred” or “pixelated” edges. Does this have anything to do with Lottie rasterizing track mattes? 
![image](https://user-images.githubusercontent.com/58711145/95390818-f3fcad80-08c3-11eb-9dbf-6c85db8a4d8e.jpeg)

My project setup is simple. Here is the [link](https://drive.google.com/file/d/1z0R5-OWuw6WU_SbZJ3J9LaRulA21ucqX/view?usp=drivesdk) if you want to check it out. The screenshots are from a different animation but the project setup is still the same. 

Here is the animation [link](https://lottiefiles.com/share/kl7ldbwc) 

Here is another screenshot from another animation using an alpha inverted matte layer with the same issue. ![image](https://user-images.githubusercontent.com/58711145/95390873-10004f00-08c4-11eb-9e0d-2f3ef7499881.jpeg)

I’m including this screenshot because I just encountered this issue recently as well. I am using alpha inverted matte layers in a precomp that is using another layer as a track matte as well. ![image](https://user-images.githubusercontent.com/58711145/95390890-168ec680-08c4-11eb-9838-95ec78d346fd.jpeg)

Here's how it supposed to look

![image](https://user-images.githubusercontent.com/58711145/95390915-20b0c500-08c4-11eb-8eed-02eafb72a141.jpeg)

**Comment by ision-industries** [comment]
I did some more tests with these animations on another iOS lottie player called Lottie Samples and I think it's worth noting my results here. 

Here is a screenshot of the line globe icon animation test. The alpha inverted matte layer does not display the pixelated edges
![Internet health Lottie Samples Screenshot](https://user-images.githubusercontent.com/58711145/95463180-77f77980-0946-11eb-850c-38264e6d3337.png)

Here is a screenshot of the last image in my post from last night. The animation plays fine on the Lottie Samples app. 
![Carousel Lottie Samples App screenshot](https://user-images.githubusercontent.com/58711145/95463421-ba20bb00-0946-11eb-858a-022fe72cc8dc.png)

So it seems to me like maybe the LottieFiles iOS app has a bug where it can't render alpha inverted mattes correctly? The chat icon project file I linked relates to all of the other screenshots I posted.
